### ISSUE TITLE
[Help] How to set SelectionBrush, CaretBrush in XAML

### ISSUE THREAD

**Issue by davidtimovski** [AvaloniaUI/AvaloniaEdit#241]
How do I style the `CaretBrush`, `SelectionBrush`, and `SelectionForeground` in XAML? I noticed `SelectionBrush` and  `SelectionForeground` are properties of `TextEditor.TextArea` and `CaretBrush` is a property of `TextEditor.TextArea.Caret` but I'm unsure how to bind them in XAML. Probably a basic lack of XAML knowledge on my side.

Previously I was using `TextBox` and I bound to its `CaretBrush` and `SelectionBrush` properties.

Might be a good idea to create some labels for Help Wanted because the docs for this library and AvalonEdit are pretty scarce.

**Comment by zflemingg1** [comment]
Did you ever get an answer to this?

**Comment by davidtimovski** [comment]
Nope. I have found zero docs on how to alter any of the styles on this thing.

**Comment by zflemingg1** [comment]
I found this useful

https://github.com/AvaloniaUI/AvaloniaEdit/issues/199

Shows how to set the selectionbrush for the text area
